### PR TITLE
First commits

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request consists of the initial functionality for this repository.

## 💭 Motivation and context ##

We need to assign some COOL AWS users to IAM groups that allow them to assume roles to send emails via SES and/or manage the SES email suppression list.  This pull request resolves cisagov/cool-system#195.

## 🧪 Testing ##

I applied these changes to our production COOL environment.  I have also requested that the four P&E folks currently in the groups that have been created retest their environments and confirm that they can assume both roles.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,38 +2,19 @@
...
-## Examples ##
-
-* [Deploying into the default VPC](https://github.com/cisagov/cool-ses-send-email-iam/tree/develop/examples/default_vpc)
+- [Terraform](https://www.terraform.io/) installed on your system.
+- An accessible AWS S3 bucket to store Terraform state
+  (specified in [backend.tf](backend.tf)).
+- An accessible AWS DynamoDB database to store the Terraform state lock
+  (specified in [backend.tf](backend.tf)).
+- Access to all of the Terraform remote states specified in
+  [remote_states.tf](remote_states.tf).
```
Comment (dav3r) [inline README.md:17]: Swiped from https://github.com/cisagov/cool-certificate-manager-iam#cool-certificate-manager-iam
Please check to see if any of our other Terraform projects should be on this list.
```suggestion
  [remote_states.tf](remote_states.tf).
  - User accounts for all users must have been created previously.  We
  recommend using the
  [`cisagov/cool-users-non-admin`](https://github.com/cisagov/cool-users-non-admin)
  repository to create users.
  - Terraform in
  [`cisagov/cool-dns-cyber.dhs.gov`](https://github.com/cisagov/cool-dns-cyber.dhs.gov)
  must have been applied.
```

**Review by dav3r — APPROVED** [review]
This looks great to me! 📫 
Please note my one small change request in the README.

File: policies.tf [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# The policy document that allows assumption of the email sending role
+# in the DNS account.
+data "aws_iam_policy_document" "email_sending" {
+  statement {
+    actions = [
+      "sts:AssumeRole",
```
Comment (mcdonnnj) [inline policies.tf:6]:
```suggestion
      "sts:AssumeRole",
      "sts:TagSession",
```

File: policies.tf [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  policy      = data.aws_iam_policy_document.email_sending.json
+  tags        = var.tags
+}
+
+# The policy document that allows assumption of the email suppression
+# list management role in the DNS account.
+data "aws_iam_policy_document" "suppression_list_management" {
+  statement {
+    actions = [
+      "sts:AssumeRole",
```
Comment (mcdonnnj) [inline policies.tf:32]:
```suggestion
      "sts:AssumeRole",
      "sts:TagSession",
```

**Review by mcdonnnj — APPROVED** [review]
Thanks for getting this together. I had a couple suggestions about additional permissions, but LGTM otherwise ✔

**Pull request merged**